domain/lib/src/model/sharedspace/versioning_parameter.dart

```diff
@@ -37,7 +37,7 @@ class VersioningParameter extends Equatable {
   VersioningParameter(this.enable);
 
   factory VersioningParameter.initial() {
-    return VersioningParameter(false);
+    return VersioningParameter(true);
```

Comment:
why we need this change?

Reply:
That is the default value when the returned shared space object has no VersioningParameter value. I checked the web version and found that the initial setting is like that. I also don't understand why they put it like that for this object. Even so, I have checked that setting `true` or `false` has no effect